arci-ros/src/ros_nav_client.rs

```diff
@@ -154,7 +154,7 @@ impl RosNavClient {
         Ok(())
     }
 
-    fn wait_until_reach(&self, goal_id: &str, timeout: std::time::Duration) -> Result<(), Error> {
+    fn wait_until_reach(&self, goal_id: &str, timeout: std::time::Duration) {
```

Comment:
By the way, this function always return Ok before this change.
I think it's a bug. (not clear)